clients/cli/src/main/java/org/apache/gravitino/cli/commands/AddOptionalColumn.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.gravitino.cli.commands;
+
+import org.apache.gravitino.NameIdentifier;
+import org.apache.gravitino.cli.DefaultConverter;
+import org.apache.gravitino.cli.ErrorMessages;
+import org.apache.gravitino.cli.ParseType;
+import org.apache.gravitino.cli.PositionConverter;
+import org.apache.gravitino.client.GravitinoClient;
+import org.apache.gravitino.exceptions.NoSuchCatalogException;
+import org.apache.gravitino.exceptions.NoSuchMetalakeException;
+import org.apache.gravitino.exceptions.NoSuchSchemaException;
+import org.apache.gravitino.exceptions.NoSuchTableException;
+import org.apache.gravitino.rel.TableChange;
+import org.apache.gravitino.rel.types.Type;
+
+public class AddOptionalColumn extends Command {
```

Comment:
What does "optional" here mean? In my understanding, "AddColumn" is good to describe the behavior.

Reply:
It was following existing naming but perhaps that has changed since this was written? The Java API still says "Create a TableChange for adding an optional column." I guess that all columns added need to be optional as they could not be added if the table has existing data.

Reply:
renamed to AddColumn